test/method.getEmailDetails.test.js

```diff
@@ -31,7 +31,8 @@ tap.test('getEmailDetails - with yaml', (assert) => {
     data: {
       firstName: 'bob',
       lastName: 'smith'
-    }
+    },
+    demo: 1 // disable the tracking pixel and uuid generation
```

Comment:
This is "originally" a flag to disable features that you wouldn't need in either the view or in test (like tracking). I changed the flag to be more specific: `disableTracking` that you can add to the payload to override the configured settings.